swagger/current/swagger.json

```diff
@@ -315,119 +315,6 @@
         }
       }
     },
-    "/communications/sync": {
```

Comment:
Can we raise a separate PR with the swagger changes for communications?

Reply:
I'll run swaggerize on master and create a separate PR.
this should go once we merge that.

Reply:
https://github.com/simpledotorg/simple-server/pull/432